PATCH-P0-SUSE: Fix for logrorate config

This patch fixes the logrotarte config file for mysql.
Read more at https://www.novell.com/support/kb/doc.php?id=7005219

Index: support-files/mysql-log-rotate.sh
===================================================================
--- support-files/mysql-log-rotate.sh.orig
+++ support-files/mysql-log-rotate.sh
@@ -21,6 +21,7 @@
 
 /var/log/mysql/mysqld.log {
         # create 600 mysql mysql
+        su mysql mysql
         notifempty
         daily
         rotate 5
